OracleAccessManagement/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Author: Kaushik C
+#
+# Description: script to build a Docker image for Oracle Access Manager 
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+#
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] -d [domain-type] [-s]
+Builds a Docker Image for Oracle OAM Suite .
+
+Parameters:
+   -h: view usage
+   -v: Release version to build. Required. E.g 12.2.1.4.0
+   -d: Domain type to be built. Required for building the domain oam
+   -s: skips the MD5 check of packages (DEFAULT)
+
+LICENSE CDDL 1.0 + GPL 2.0
+Copyright (c) 2016-2017: Oracle and/or its affiliates. All rights reserved.
+EOF
+exit 0
+}
+
+
+# Validate packages
+checksumPackages() {
+  echo "Checking if required packages are present and valid..."
+  md5sum -c *.download
+  if [ "$?" -ne 0 ]; then
+    echo "MD5 for required packages to build this image did not match!"
+    echo "Make sure to download missing files in folder dockerfiles. See *.download files for more information"
+    exit $?
+  fi
+}
+
+#Parameters
+VERSION="12.2.1.4.0"
+SKIPMD5=1
+while getopts "hsdgiv:t:" optname; do
+  case "$optname" in
+    "h")
+      usage
+      ;;
+    "s")
+      SKIPMD5=1
+      ;;
+    "t")
+      DOMAINTYPE="$OPTARG"
+      echo "Domain Type selected is : "$DOMAINTYPE
+      ;;
+    "v")
+      VERSION="$OPTARG"
+      ;;
+         *)
+    # Should not occur
+      echo "Unknown error while processing options inside buildDockerImage.sh"
+      ;;
+  esac
+done
+
+# OAM Image Name
+IMAGE_NAME="oracle/oam:$VERSION"
+DOCKERFILE_NAME="Dockerfile"
+
+
+# Go into version folder
+cd $VERSION
+echo  "version --> $VERSION  "
+
+# Proxy settings
+PROXY_SETTINGS="--build-arg http_proxy=http://www-proxy.us.oracle.com:80"
```

Comment:
Never include internal Oracle hostnames in public code. This must be removed ASAP.

Reply:
Removed.